packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -354,6 +348,13 @@ export interface IGCRuntimeOptions {
 	 */
 	sessionExpiryTimeoutMs?: number;
 
+	/**
+	 * If true, then Tombstoned objects will merely trigger a log message when loaded.
+	 *
+	 * By default, attempting to load a Tombstoned object will result in an error.
+	 */
+	disableGCThrowOnTombstoneLoad?: boolean;
```

Comment:
Yeah I don't think we need this. A config setting should be enough.

Reply:
As discussed offline - On second thought, I'm going to leave it for now.  We are at the pivotal few months where key partners are enabling Tombstone, and if there are any issues I don't want additional friction.  The GC Option is the most ergonomic for them.  I'll plan to remove it a month or two later.